frontend/src/scenes/onboarding/sdks/SDKSnippet.tsx

```diff
@@ -12,6 +20,64 @@ export const SDKSnippet = ({ sdk, sdkInstructions }: { sdk: SDK; sdkInstructions
                 </Link>
             </div>
             <div className="deprecated-space-y-4">{sdkInstructions()}</div>
+            <div className="deprecated-space-y-4">
+                {/* Only show survey announcements in non surveys product context */}
+                {productKey !== ProductKey.SURVEYS && (
```

Comment:
i think it only makes sense for supported mobile products as well.
eg session replay, feature flags, experiments, prod analytics and surveys.
i dont think eg dataware house would benefit from this mobile ad

Reply:
Ah then it makes a lot of sense.
What i meant is if its a product that theres no mobile support, and they are installing a non mobile sdk eg node, this should not be shown, if thats the case, all good